invenio/modules/pages/admin.py

```diff
@@ -60,6 +62,7 @@ class PagesAdmin(ModelView):
 
     form_args = dict(
         template_name=dict(
+            default="default.html",
```

Comment:
Hmm, I'm not sure how to do it. @jirikuncar, do you know how could this be solved?